File: pipe & validation ng/app/dto/employee-basic-dto.ts
export class EmployeeBasicDTO {
  empId: number;
  employeeName: string;

  designation: string;
  location: string;
  panNumber: string;
  projectName: string;
  verificationStatus: string;

  constructor(
    empId: number,
    employeeName: string,

    designation: string,
    location: string,
    panNumber: string,
    projectName: string,
    verificationStatus: string
  ) {
    this.empId = empId;
    this.employeeName = employeeName;

    this.designation = designation;
    this.location = location;
    this.panNumber = panNumber;
    this.projectName = projectName;
    this.verificationStatus = verificationStatus;
  }
}

File: logging/HrApp/src/main/java/com/cg/hrapp/dto/EmployeeBasicDTO.java
package com.cg.hrapp.dto;

import org.springframework.stereotype.Component;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Component
@Data
@AllArgsConstructor
@NoArgsConstructor
public class EmployeeBasicDTO {

	private String employeeName;
	private int empId;
	private String designation;
	private String location;
	private String projectName;
	private String panNumber;
	private String verificationStatus;
	
}
